frontend2/.gitignore

```diff
@@ -11,6 +11,9 @@
 # production
 /build
 
+# auto-generated files
+/src/utils/types
```

Comment:
we shouldn't add this change, we want these types to stay in our git repo

Reply:
Oh whoops that's right this should be in `.prettierignore` 🤦